libbeat/docs/tab-widgets/enable-modules-widget.asciidoc

```diff
@@ -0,0 +1,114 @@
+++++
+<div class="tabs">
+  <div role="tablist" aria-label="Enable modules">
+    <button role="tab"
+            aria-selected="true"
+            aria-controls="deb-tab-enable-modules"
+            id="deb-enable-modules">
+      DEB
+    </button>
+    <button role="tab"
+            aria-selected="false"
+            aria-controls="rpm-tab-enable-modules"
+            id="rpm-enable-modules"
+            tabindex="-1">
+      RPM
+    </button>
+    <button role="tab"
+            aria-selected="false"
+            aria-controls="mac-tab-enable-modules"
+            id="mac-enable-modules"
+            tabindex="-1">
+      MacOS
+    </button>
+    <button role="tab"
+            aria-selected="false"
+            aria-controls="brew-tab-enable-modules"
+            id="brew-enable-modules"
+            tabindex="-1">
+      Brew
+    </button>
+    <button role="tab"
+            aria-selected="false"
+            aria-controls="linux-tab-enable-modules"
+            id="linux-enable-modules"
+            tabindex="-1">
+      Linux
+    </button>
+    <button role="tab"
+            aria-selected="false"
+            aria-controls="win-tab-enable-modules"
+            id="win-enable-modules"
+            tabindex="-1">
+      Windows
+    </button>
+  </div>
+  <div tabindex="0"
+       role="tabpanel"
+       id="deb-tab-enable-modules"
+       aria-labelledby="deb-enable-modules">
+++++
+
+include::enable-modules.asciidoc[tag=deb]
+
+++++
+  </div>
+  <div tabindex="0"
+       role="tabpanel"
+       id="rpm-tab-enable-modules"
+       aria-labelledby="rpm-enable-modules"
+       hidden="">
+++++
+
+include::enable-modules.asciidoc[tag=rpm]
+
+++++
+  </div>
+  <div tabindex="0"
+       role="tabpanel"
+       id="mac-tab-enable-modules"
+       aria-labelledby="mac-enable-modules"
+       hidden="">
+++++
+
+include::enable-modules.asciidoc[tag=mac]
+
+++++
+  </div>
+  <div tabindex="0"
+       role="tabpanel"
+       id="brew-tab-enable-modules"
+       aria-labelledby="brew-enable-modules"
+       hidden="">
+++++
+
+include::enable-modules.asciidoc[tag=brew]
+
+++++
+  </div>
+  <div tabindex="0"
+       role="tabpanel"
+       id="linux-tab-enable-modules"
+       aria-labelledby="linux-enable-modules"
+       hidden="">
+++++
+
+include::enable-modules.asciidoc[tag=linux]
+
+++++
+  </div>
+  <div tabindex="0"
+       role="tabpanel"
+       id="win-tab-enable-modules"
+       aria-labelledby="win-enable-modules"
+       hidden="">
+++++
+
+include::enable-modules.asciidoc[tag=win]
+
+++++
+  </div>
+</div>
+++++
+
+include::stuff.asciidoc[]
```

Comment:
What about just including this once in `index.asciidoc`?

Reply:
I think we should add it the stylesheet for all the guides. Just parking it in an asciidoc file for now. I thought the css would need to be included for each html page, though I didn't test the index as you suggest.